environment.yml

```diff
@@ -1,16 +1,16 @@
-name: qibochem
-channels:
-- conda-forge
-- psi4
-- defaults
-dependencies:
-- python=3.9
-- psi4
-- numpy
-- sympy
-- scipy
-- pip
-- pip:
-  - openfermion
-  - qibo
-  - pyscf
+name: qibochem
+channels:
+- conda-forge
+- psi4
+- defaults
+dependencies:
+- python=3.9
+- psi4=1.7
+- numpy
+- sympy
+- scipy
+- pip
+- pip:
+  - openfermion

```

Comment:
Is it possible to add something like `openfermion>=1.5` here?

Reply:
1. Whoops, forgot about the readme, but migrating to poetry will probably change how to install, I'll go look it up and make the changes.
2. My bad, I'll toss it into the .gitignore, if there's any other common temp files that you've seen get uploaded like this, feel free to add it to the .gitignore.
3. Yes